src/main/scala/com/ibm/couchdb/TypeMapping.scala

```diff
@@ -16,7 +16,11 @@
 
 package com.ibm.couchdb
 
-final class TypeMapping private(val types: Map[String, String])
+final class TypeMapping private(val types: Map[String, String]) {
```

Comment:
Should we then make the types field private and use the forType method everywhere instead?